docs/Superoperator representations.md

```diff
@@ -12,49 +16,565 @@ where $i$ is a row and $j$ is a column index.
+
+$$\mathcal Q_{m,\mu;n,\nu} = \langle m, \mu |\mathcal Q |n,\nu \rangle $$ 
+where $|n,\nu\rangle = |n\rangle \otimes |\nu\rangle$, $m,n\in \{0,\ldots, d_A-1\}$, $\mu,\nu\in \{0,\ldots, d_B-1\}$ and all vectors are in the standard basis.
+
+With respect to these indicies some useful operations are [GRAPTN]:
+ 
+Transpose $T$: $\mathcal Q_{m,\mu;n,\nu} \mapsto \mathcal Q_{n,\nu;m,\mu},$  
+SWAP: $\mathcal Q_{m,\mu;n,\nu} \mapsto \mathcal Q_{\mu,m;\nu,n},$  
+Row-reshuffling $R_r$: $\mathcal Q_{m,\mu;n,\nu} \mapsto \mathcal Q_{m,n;\mu,\nu},$  
+Col-reshuffling $R$: $\mathcal Q_{m,\mu;n,\nu} \mapsto \mathcal Q_{\nu,\mu;n,m}.$
+
+The importance of understanding reshuffling can be understood as understanding the relationship between 
+$${\rm vec}(G)\otimes {\rm vec}(\Gamma) \quad {\rm and} \quad  {\rm vec}(G\otimes\Gamma)$$
+where $G$ and $\Gamma$ are matrices that act on $\mathbb H_A$ and $\mathbb H_B$ respectively, as explaind in [VECQO].
+
+### A note on numerical implementations
+Most linear algebra (or tensor) libraries have the ablity to `reshape` a matrix and `swapaxes` (or sometimes it is called `permute_dims`). 
+
+If you are trying to reshuffle indicies usually the first job is to write your matrix in tensor form. This requires reshaping a $d_A^2\times d_B^2$ matrix into a $d_A\times d_A\times d_B \times d_B$ tensor. Next you `permute1_dims` or `swapaxes`. Often $d_A = d_B$ so we `reshape` to a Matrix that has the same dimensions as the orignal  $d_A^2\times d_A^2$ matix.
+
+
+## The $n$-qubit Pauli basis 
+ The $n$-qubit Pauli basis is denoted $\mathcal P^{\otimes n} $ where $\mathcal  P = \{ I, X, Y, Z \}$ are the usual Pauli matricies. It is an operator baisis for the $d = 2^n$ dimensional Hilbert space and there are $d^2 = 4^n$ operators in $\mathcal P^{\otimes n} $. If one divides all the operators by $\sqrt{d}$ the basis is orthonormal with respect to the Hilbert-Schmidt inner product.
+ 
+It is often convenient to index the $d^2$ operators with a single label, e.g. $P_1=I^{\otimes n},\, \ldots,\, P_{d^2}= Z^{\otimes n}$  (or $P_0=I^{\otimes n}$ if you like zero indexing). In anycase, as these operators are Hermitian and unitary they obey $P_i^2=I^{\otimes n}$.
+ 
+To be explicit, for two qubits $d=4$ and we have 16 operators e.g. $\{II, IX, IY, IZ, XI, XX, XY, ..., ZZ\}$ were $II$ should be intepreted as $I\otimes I$ etc. The single index would be $\{P_1, P_2, P_3, P_4, P_5, P_6, P_7, ..., P_{16}\}$.
+
+## Quantum channels in the Kraus decomposition (or operator-sum representation)
+A completely positive map on the state $\rho$ can be written using a set of Kraus operators $\{ M_k \}$ as
+
+
+$$\rho' =\mathcal E (\rho) = \sum_{k=1}^N M_k \rho M_k^\dagger, $$
+where $\rho'$ is the state at the output of the channel.
+
+If $\sum_k M_k^\dagger M_k= I $ the map is trace preserving. It turns out that $N\le d^2$ where $d$ is the Hilbert space dimension e.g. $d=2^n$ for $n$ qubits.
+
+
+## Kraus to process matrix (aka $\chi$ or chi matrix)
```

Comment:
It is important to point out that the chi matrix is the same thing as the Choi matrix.  Typically the Choi matrix is defined in the computational basis, while the chi matrix uses the Pauli basis.  Moreover, they may have different normalization conventions.

Reply:
I would also stay away from talking about a "process matrix" -- it is kinda ambiguous, since the process can be represented by many different matrices.

Reply:
In the text have not really described the relationships between the various representations. I was hoping the reader would look at the references. I'll point out the basis change / reshuffling arguments.

As you know process matrix is pretty standard terminology. But I'll change it to chi matrix and make the process parenthetical. I'll need to do this in the code too 

- [x] fix code process -> chi